### ISSUE TITLE
Question about training on Human Dataset

### ISSUE THREAD

**Issue by WinstonDeng** [nianticlabs/monodepth2#224]
Hi, I'm trying to train your model on human dataset, e.g., H3.6M, KITTI-Person, or my videos. Even though training with the same params (self.full_res_shape has been modified) on KITTI-Person, my visual results are far from the performance of **Additional Wander Results** in your paper.

My visual results:

![image](https://user-images.githubusercontent.com/38663473/88145627-4fa48c80-cc2d-11ea-9419-a760da81f558.png)
![image](https://user-images.githubusercontent.com/38663473/88145657-5af7b800-cc2d-11ea-8534-fa0eb620f610.png)

The loss decreased slowly and was close to 0.009.

Could you give me some advice or share your pre-trained model on Wander Dataset?

**Comment by sushy446** [comment]
I ran into the same problem.
I have trained on my own monocular data set(4794 training items and 1005 validation items), but the result is worse. The self.K and self.full_res_shape have been changed, what did I do wrong? Thanks a lot!
![000685](https://user-images.githubusercontent.com/68721241/88367367-2f580780-cdbe-11ea-97f2-46876b7cab5b.jpg)
![000685_disp](https://user-images.githubusercontent.com/68721241/88367392-38e16f80-cdbe-11ea-9899-2c40f954425f.jpeg)

**Comment by mrharicot** [comment]
@WinstonDeng It is very difficult to figure out what went wrong as I am not familiar with these datasets except the KITTI-Persons which is particularly bad as it is a single scene without moving camera which is close to the worst case scenario for training self supervised depth.

@sushy446  It is also very difficult to figure out what went wrong in your case, the camera shows lens distortion which breaks the pinhole camera model used by monodepth2. You also have static text on the images which tends to throw off the network if not ignored. Finally 5k training samples is very little. Also please open a separate issue as it is unrelated to this one.

**Comment by WinstonDeng** [comment]
> @WinstonDeng It is very difficult to figure out what went wrong as I am not familiar with these datasets except the KITTI-Persons which is particularly bad as it is a single scene without moving camera which is close to the worst case scenario for training self supervised depth.
> 
> @sushy446 It is also very difficult to figure out what went wrong in your case, the camera shows lens distortion which breaks the pinhole camera model used by monodepth2. You also have static text on the images which tends to throw off the network if not ignored. Finally 5k training samples is very little. Also please open a separate issue as it is unrelated to this one.

Thanks for your reply! 
Agree with your analysis, it is a possible reason that static pixels are ignored by masking operation.  The results on Walking Youtube Video with a hand-camera view looks normal.
